src/review/Post.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import './Post.scss';
+import { AiOutlineArrowUp, AiOutlineArrowDown } from 'react-icons/ai';
+
+const productInfo = [
+  {
+    img: 'https://crowcanyon.co.kr/web/product/tiny/202201/340ff70c75de7d1e91b0979a59a228e6.jpg',
+    NEW: true,
+    품절: true,
+    name: 'D49 스몰스푼 베이비 블루마블',
+    price: 9000,
+  },
+];
+
+const Post = () => {
+  return (
+    <div className="reviewContainer">
+      <header className="slideRight">
+        <h1>상품후기</h1>
+      </header>
+      <section className="product">
+        <div className="productInfo">
+          <img src={`${productInfo[0].img}`} alt="productPhoto" />
+          <div className="detailedInfo">
+            <div className="receivingStatus">
+              <span>{productInfo[0].NEW ? 'NEW' : null}</span>
+              <span>{productInfo[0].품절 ? '품절' : null}</span>
```

Comment:
```suggestion
              <span>{productInfo[0].NEW && 'NEW'}</span>
              <span>{productInfo[0].품절 && '품절'}</span>
```
삼항연산자 말고 && 연산자로도 표현할 수 있어보이네요!

Reply:
수정했지만 이해를 못했습니다 ㅠㅠ